icicle-core/src/it/scala/com/intenthq/icicle/IcicleIdGeneratorIntegrationSpec.scala

```diff
@@ -10,7 +10,7 @@ import org.specs2.specification.Scope
 
 import scala.collection.JavaConversions._
 
-class IcicleIdGeneratorIntegrationSpec extends Specification {
+object IcicleIdGeneratorIntegrationSpec extends Specification {
```

Comment:
Swapped the `class` for `object` on all specs, this was a weird thing and I don't know what it was done.